server/src/types/resolvers/Channel/mutation.ts

```diff
@@ -63,23 +62,17 @@ export const createChannel = mutationField('createChannel', {
         },
       });
 
-    const createMemberships = async (channelId: string, userIds: string[]) => {
-      const promises = [];
-
-      for (const userId of userIds) {
-        const batch = ctx.prisma.membership.create({
-          data: {
+    const createMemberships = async (channelId: string, userIds: readonly string[]) => {
+       const promises: any[] = userIds.map((userId) => ctx.prisma.membership.create({
```

Comment:
Please do not use `any`. Try to find any workaround to this.

Reply:
I think It would be better to use `Promise<Membership>[]` instead of `any[]`. `Membership` type can be imported from `@prisma/client` like `import { Membership } from '@prisma/client';`.

Reply:
> I think It would be better to use `Promise<Membership>[]` instead of `any[]`. `Membership` type can be imported from `@prisma/client` like `import { Membership } from '@prisma/client';`.

Thanks for the advice, I think it would be nice to do that.